sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/providers/BigQueryWriteConfiguration.java

```diff
@@ -194,6 +194,9 @@ public static Builder builder() {
           + "Is mutually exclusive with 'keep' and 'drop'.")
   public abstract @Nullable String getOnly();
 
+  @SchemaFieldDescription("A list of columns to cluster the BigQuery table by.")
+  public abstract @Nullable List<String> getClustering();
```

Comment:
nit: what do you think of the name `getClusteringFields()`?

We mainly thinking about the generated parameter name for ManagedIO and Xlang: `clustering` vs `clustering_fields`